packages/migration/src/migrations/hearth/20230905130632-elastic-search-property-rename.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+
+import { updateFieldNameByCompositionId } from '../../utils/elasticsearch-helper.js'
+
+export const up = async () => {
+  // rename field name contactRelationship to informantType on elasticSearch
+  await updateFieldNameByCompositionId('contactRelationship', 'informantType')
```

Comment:
@Nil20 If the field contactRelationship is being renamed informantType, then the parameters should be the other way round if you look at the function updateFieldNameByCompositionId

Reply:
@Nil20 Since this function is currently only being used in packages/migration/src/migrations/hearth/20221110092434-add_identifiers_DOB_gender_location.ts, it's worth changing the signature `updateFieldNameByCompositionId(newValue, oldValue)` to 

`renameField(oldValue, newValue)`

Notice both changed name and flipped arguments.